.shellcheck.yaml

```diff
@@ -0,0 +1,119 @@
+ignored:
+  - SC2140
```

Comment:
can you comment these or at the very least have a comment pointing to a list of what all of these are?